Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/ui/consent/FamilyTestConsentFragment.kt

```diff
@@ -63,6 +65,38 @@ class FamilyTestConsentFragment : Fragment(R.layout.fragment_family_test_consent
             }
         }
 
+        viewModel.registrationState.observe2(this) { state ->
```

Comment:
The state of the button is handled by isValid, don't do this here too. Rather adjust the isValid logic inside the viewmodel.

Reply:
refactored isValid logic into isSubmittable State